src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -524,7 +524,7 @@ public function handleException(\Throwable $exception): void
         $exception = $this->enhanceError($exception);
 
         $exceptionHandler = $this->exceptionHandler;
-        $this->exceptionHandler = [$this, 'renderException'];
+        $this->exceptionHandler = $this->renderException(...);
```

Comment:
Does the check in the next line still work after this change?

Reply:
Oh, I wasn't aware of this behavior and the fact it is not the same reference behind. Thanks! I might add a test case for this later, it wasn't detected by unit tests